make minimim_protocol_version configurable

jschaul opened the pull request:
Make minimim_protocol_version configurable.

(I wasn't able to run `make readme` to completion as this repo and/or the included build harness fails due to presupposed binaries (`/bin/bash: gomplate: command not found`) whose origins are not documented)

osterman commented:
@jschaul sorry about the trouble.

Run `make readme/deps` to download gomplate. I'll see about making that process smoother.

osterman commented:
@jschaul, sorry we missed your PR. Join our slack team and you can reach us directly in #terraform channel.

[![Slack Community](https://slack.cloudposse.com/badge.svg)](https://slack.cloudposse.com)

aknysh commented:
@jschaul  looks good, thanks.
please run `make readme/deps` and then `make readme` to update `README`.

jschaul commented:
Updated and rebased. Should be good to be merged now.

(Your readme regeneration process is slightly clunky; I had to manually commit only parts of the generated readme, since that build harness doesn't update automatically, in order to avoid undoing the changes introduced by #21)

osterman commented:
@jschaul thanks for the feedback. Always looking to improve the process. There should be zero need to commit certain parts (haven't run into that before). If you can share screenshots, perhaps it will help me understand the problem.

To update the build-harness, just re-run `make init`

Open to suggestions. Don't think we automatically want to update the build-harness. 

 We recently went through the process of updating all ~120 modules with our `README.yaml` spec.
